.github/workflows/canary-integration-test.yml

```diff
@@ -57,6 +57,11 @@ jobs:
         kubectl -n rook-ceph exec $toolbox -- mkdir -p /etc/ceph/test-data
         kubectl -n rook-ceph cp tests/ceph-status-out $toolbox:/etc/ceph/test-data/
         kubectl -n rook-ceph cp deploy/examples/create-external-cluster-resources.py $toolbox:/etc/ceph
+        # remove the existing client auth that will be re-created by external script
+        kubectl -n rook-ceph exec $toolbox -- ceph auth del client.csi-cephfs-node
+        kubectl -n rook-ceph exec $toolbox -- ceph auth del client.csi-cephfs-provisioner
+        kubectl -n rook-ceph exec $toolbox -- ceph auth del client.csi-rbd-node
+        kubectl -n rook-ceph exec $toolbox -- ceph auth del client.csi-rbd-provisioner
         timeout 10 sh -c "until kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --rbd-data-pool-name replicapool; do echo 'waiting for script to succeed' && sleep 1; done"
 
```

Comment:
Let's print the list of caps after this run so we have a good idea of what's being done.

Reply:
sure